metakb/cli.py

```diff
@@ -0,0 +1,171 @@
+"""
+Provide CLI utility for performing data collection, transformation, and upload
+to graph datastore.
+"""
+import click
+from os import environ
+from metakb.database import Graph
+from metakb.harvesters import CIViC, MOAlmanac
+from metakb.transform import CIViCTransform, MOATransform
+import logging
+from metakb import PROJECT_ROOT
+from disease.database import Database as DiseaseDatabase
+from disease.schemas import SourceName as DiseaseSources
+from disease.cli import CLI as DiseaseCLI
+from therapy.database import Database as TherapyDatabase
+from therapy import ACCEPTED_SOURCES as TherapySources
+from therapy import SOURCES as TherapySourceLookup
+from therapy.cli import CLI as TherapyCLI
+from gene.database import Database as GeneDatabase
+from gene.cli import CLI as GeneCLI
+
+
+logger = logging.getLogger('metakb')
+logger.setLevel(logging.DEBUG)
+
+
+class CLI:
+    """Update database."""
+
+    @click.command()
+    @click.option(
+        '--db_url',
+        help=('URL endpoint for the application database. Can also be '
+              'provided via environment variable METAKB_DB_URL.')
+    )
+    @click.option(
+        '--db_username',
```

Comment:
Above, you included that you could use environment variable. I think you should do these here and for `--db_password`.